golang.go

```diff
@@ -0,0 +1,19 @@
+// package docummentation
```

Comment:
Typo inside comment in word `docummentation`. Possible corrections: documentation (85),